rulebook.tex

```diff
@@ -612,26 +670,182 @@ \subsection{Machines}
+  in the working height of some or all parts of a station.}.
+Working space between guiding lanes is \SI{4.5}{\centi\metre}. Setup lanes and
+shelves feature approximately the same space for handling and
+adjusting. All diffuse sensors (input side) have been removed to allow
+for infrared-emitting cameras.  To simplify the delivery on the belt,
+each machine will be equipped with a narrowing cone on its input side
+(\reffig{fig:narrow-cone}).  You can order them from Festo Didactic SE
+(C. Deppe) or download the STL model file from the \ac{RCLL} site.
+\begin{figure}
+    \includegraphics[height=8cm]{narrowConeSketch.jpg}
+\caption{Narrowing cone}
+\label{fig:narrow-cone}
+\end{figure}
+% Still unclear, thus commenting out.
+% If observed from the output side of machines (where the signal unit
+% is placed) the conveyor space opens up at half of the aluminum
+% profile thus starting at \SI{0,35}{\metre}.
+%\begin{figure}[b!]
+%    \centering
+%    \includegraphics[height=7cm]{MPS2015/MPS_Layout.jpg}
+%
+%    \caption{General Machine Layout}
+%    \label{fig:MPS-Layout}
+%\end{figure}
+
+\subsubsection{Machine Positioning}
+\label{sec:machine-swapping}
+The zones of each field are named according to the following scheme:
+A team prefix (``C-'' for cyan, ``M-'' for
+magenta), ``Z'' to distinguish zone names, and a grid coordinate form the name
+of a zone.
+For example, The zone C-Z23 is a zone on the cyan primary half, which is
+the second along the positive half of the X-axis and the third along the Y-axis.
+Zones will not be physically represented or visible on the field.
+They cannot be used for any other purpose than machine positioning.
+
+The zones and rotations for the \ac{MPS} will be randomly chosen by the
+\ac{refbox}.
+Each game will have a new randomly generated field layout, the referees will
+place the \ac{MPS} in the middle of the zone with given rotation at there
+best effort but errors of the positioning should be expected.
+
+A \ac{MPS} can generally  be placed in any zones except those at the
+entrance of the insertion area (the light red areas in
+\reffig{fig:competition-area} and \reffig{fig:competition-area-challenges}).
+These zones are needed to enter the field. Each \ac{MPS} has one out of 8
+possible orientations, starting at \ang{0} with steps of \ang{45}. To denote
+the orientation, we assign a local right-handed coordinate system to
+each \ac{MPS} station. The x-axis is given by the conveyor pointing from
+the output to the input. The origin is the center point of the
+conveyor. The orientation is then the relative rotation of the local
+\ac{MPS} system compared to the fixed field coordinate system. For example,
+at an \ac{MPS} orientation of \ang{0}, its local coordinate system is
+aligned with the field (in terms of orientation). Furthermore, an \ac{MPS}
+at an orientation of \ang{90} will be rotated counter-clockwise a
+quarter turn compared to the field system.
```

Comment:
Maybe we should create a sketch to explain this, I guess that is easier to understand than text.

Reply:
The sketch is already in Figure 1, but i think pointing to it is useful here. Also added a brief explanation